nservicebus/hosting/nservicebus-host/installation.md

```diff
@@ -20,83 +32,103 @@ NServiceBus.Host.exe [/install [/serviceName]
+/endpointConfigurationType:"QualifiedNameSpace.EndpointConfigType, AssemblyName"
+/username:"corp\serviceuser"
+/password:"p@ssw0rd!" NServiceBus.Production
 ```
 
-To set the actual name of the Windows Services in the registry, specify `/serviceName:ServiceName`. This is different from what is displayed in the Windows Service Manager.
+
+### ServiceName
+
+To set the actual name of the Windows Services in the registry, specify `/serviceName:ServiceName`. By default, the name of the service is the name of the endpoint and the endpoint name is the namespace of the endpoint configuration class.
+
+Note: The Windows Service name is different from what is displayed in the Windows Service Manager. The Windows Service Manager shows the *DisplayName*.
+
+
+### DisplayName
 
 To set the name of the Windows Service as it is displayed in the Windows Service Manager, specify `/displayName:ServiceDisplayName`.
 
-If `/displayName` is not specified, but `/serviceName` is, the display name does not become what was passed in the `/serviceName`, but rather the default described above.
+If `/displayName` is not specified, but `/serviceName` is, the display name does not become what was passed in the `/serviceName`, but the default *ServiceName* as described above is used.
+
+
+### Description
 
 To set the description shown in the Windows Service Manager, specify
 `/description:DescriptionOfService`.
 
-To install multiple instances of the same service by providing each a different instance name, use the 'instance' flag. For example: `/instance:Instance5`.
+
+### EndpointConfigurationType
+
+Specify the type implementing `IConfigureThisEndpoint` that should be used by using the `/endpointConfigurationType` parameter. The provided value needs to be the [assembly qualified type name](https://msdn.microsoft.com/en-us/library/system.type.assemblyqualifiedname(v=vs.110).aspx).
+
+
+### EndpointName
+
+Configures the name of the endpoint. By default, the endpoint name is the namespace of the endpoint configuration class.
+
+
+### ScannedAssemblies
+
+Configures NServiceBus to scan only the specified assemblies.
+
+
+### DependsOn
+
+Specifies the names of services or groups which must start before this service. Multiple services/groups are separated by a comma.
+
+
+### SideBySide
+
+By using the `/sideBySide` argument, the endpoints version will be appended to service name. This setting enables side-by-side operations by allowing multiple endpoints with different versions to run at the same time.
+
+
+### StartManually
 
 By default, Windows Services start automatically when the operating system starts. To change that, add
 `/startManually` to the `/install` command.
 
-To specify under which account the service runs, pass in the username and password of that account.
 
-Here is an example of the `/install` command line:
+### Username
+
+To specify under which account the service runs, pass in the username of that account.
```

Comment:
What will the service account default be? LocalUser? LocalService? LocalSystem? can you specify that? 

> To specify under which account the service runs, pass in the username of that account.

How about:
Provide the username that the service should execute in the context of, instead of running as [whatever the default is]